src/Components/Web.JS/src/DomWrapper.ts

```diff
@@ -22,7 +22,12 @@ function focus(element: HTMLOrSVGElement, preventScroll: boolean): void {
   }
 }
 
-function focusBySelector(selector: string): void {
+function focusOnNavigate(selector: string): void {
+  const preventScroll = location.hash.length > 1 && elementExists(location.hash.slice(1));
+  focusBySelector(selector, preventScroll);
```

Comment:
I would suggest we make this *always* use `preventScroll`, regardless of whether there's a hash in the URL. The `<FocusOnNavigate>` component was never intended to affect scroll position, and historically wouldn't do so based on how the project templates were set up.

I think it's better to just `preventScroll` in all cases here because otherwise the behavior becomes less predictable - there's a conflict between what the hash routing is trying to do, and what `<FocusOnNavigate>` would be trying to do, and it would be hard to know for sure which one the developer wants to win this fight. It will be clearer if `<FocusOnNavigate>` *only* causes focusing and never causes scrolling, then there is no ambiguity.

Reply:
I agree. I changed `focusBySelector` to always preventscroll and removed `focusOnNavigate` since there is no longer need for it. I also removed 2 test in e2e that were schecking scrolling when FocusOnNavigate selector is on the page.